sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs

```diff
@@ -56,7 +56,7 @@ public static class ResourceGroupHelper
         ///  using the TestSupport Infrastructure
         /// </summary>
         /// <param name="handler"></param>
-        /// <returns>A netowrk management client, created from the current context (environment variables)</returns>
```

Comment:
DNS is a left over question needs to be solved. We should update the readme in swagger repo, add .NET block and trying to regenerate it. For this PR, If possible, I suggest we can exclude this change for now.

Reply:
I agree;  the best path forward would be to revert this change to unblock the rest of the PR.

Reply:
thanks sir @HarveyLink  for review !, that means the best way is for me to return it as it was right?

Reply:
That's correct, @slowy07.  If you undo your changes to the following, that should pass CI and unblock getting things merged:

- `sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Generated/Models/KafkaRestProperties.cs`
- `sdk/dns/Microsoft.Azure.Management.Dns/tests/Helpers/ResourceGroupHelpers.cs`

Reply:
sorry sir @jsquire, how can undo the commit ??